lib/api/routes/iterations.rb

```diff
@@ -48,12 +48,19 @@ class Iterations < Core
           halt 401, {error: message}.to_json
         end
 
+        #rubocop code analysis start
```

Comment:
Incorrect indentation detected (column 8 instead of 9).<br>Missing space after #.